gulp-tasks/test.js

```diff
@@ -242,6 +242,26 @@ function findBadMDExtensions() {
 }
 
 /**
+ * Chec to ensure there an o JavaSCript files in the content/ folder.
+ *
+ * Note: the promise will always be fulfilled, even if it fails.
+ *
+ * @return {promise} An empty promsie
+ */
+ function findJavaSCriptFiles() {
+  return new Promise(function(resolve, reject) {
+    var contentPath = GLOBAL.WF.src.content;
+    var patterns = ['**/*.js', '!**/_code/*.js', '!**/event-map.js'];
+    var files = wfHelper.getFileList(contentPath, patterns);
+    files.forEach(function(filename) {
+      summary.files++;
+      logError(filename.filePath, '\n\tJS files must be in a directory named /_code.')
```

Comment:
Be consistent in logging, don't use the `\n\t`, instead I'd recommend: 'JavaScript files are not permitted.'